Foundation/NSJSONSerialization.swift

```diff
@@ -68,3 +86,399 @@ public class NSJSONSerialization : NSObject {
         NSUnimplemented()
     }
 }
+
+//MARK: - Deserialization
+internal extension NSJSONSerialization {
+    
+    static func JSONObjectWithString(string: String) throws -> AnyObject {
+        let parser = JSONDeserializer.UnicodeParser(viewSkippingBOM: string.unicodeScalars)
+        if let (object, _) = try JSONDeserializer.parseObject(parser) {
+            return object
```

Comment:
object in this case does not conform to AnyObject since it is an implicit conversion